auth-external-api/src/main/java/uk/gov/di/authentication/external/services/UserInfoService.java

```diff
@@ -32,13 +32,13 @@ public UserInfo populateUserInfo(AccessTokenStore accessTokenInfo) {
         String internalSubjectId = accessTokenInfo.getSubjectID();
         var userProfile = authenticationService.getUserProfileFromSubject(internalSubjectId);
 
-        String internalRpPairwiseId =
-                ClientSubjectHelper.calculatePairwiseIdentifier(
-                        internalSubjectId,
+        Subject internalRpPairwiseId =
```

Comment:
While we're here, could you rename this to remove `Rp`? I think it makes it quite unclear

Reply:
(otherwise happy for this to go in!